### PR TITLE
Use flag config-path to sanitize-prow-jobs

### PR DESCRIPTION
**Pull request by hongkailiu**
Required by https://github.com/openshift/ci-tools/pull/660#discussion_r413085427

/cc @stevekuznetsov

### CODE REVIEW

**Comment by hongkailiu** [comment]
/hold 

waiting for the image of https://github.com/openshift/ci-tools/pull/687

File: hack/order-prow-job-config.sh [hunk 1]
```diff
@@ -15,8 +15,9 @@ if [[ ! -d "${base_dir}" ]]; then
 fi
 
 ci_operator_dir="${base_dir}/ci-operator"
+config_path="${base_dir}/core-services/sanitize-prow-jobs/_config.yaml"
```
Comment (stevekuznetsov) [inline hack/order-prow-job-config.sh:18]: This is used in once place -- just add it to the `--config-path` line, no need for a new var

Comment (hongkailiu) [reply hack/order-prow-job-config.sh]: Done.

**Pull request merged**